Mathlib/LinearAlgebra/FreeModule/Finite/Rank.lean

```diff
@@ -327,45 +327,48 @@ theorem basisUnique_repr_eq_zero_iff {ι : Type*} [Unique ι]
     fun hv => by rw [hv, LinearEquiv.map_zero, Finsupp.zero_apply]⟩
 #align finite_dimensional.basis_unique.repr_eq_zero_iff FiniteDimensional.basisUnique_repr_eq_zero_iff
 
-theorem cardinal_mk_le_finrank_of_linearIndependent [Module.Finite R V]
+end FiniteDimensional
+
+namespace LinearIndependent
+
+theorem cardinal_mk_le_finrank [Module.Finite R V]
```

Comment:
Similar to a [previous commit](https://github.com/leanprover-community/mathlib4/pull/9046/commits/0478370c1e2c9c6212f8f714fd2503f7094c80ad) I've changed lemma names to make available LinearIndependent dot notations in this file as well (including some new lemmas introduced by @erdOne in #8872). If I split these as a dependent PR it could reduce "files changed" by 4~5 files.

Reply:
Opened #9144
@eric-wieser Do you think I need to split this PR further down?